### PR TITLE
Replaced hardcoded url's with variable calls so that we can define wh…

### PR DESCRIPTION
**Pull request by nigelgbanks**
…ere karaf is install.

### CODE REVIEW

**Comment by nigelgbanks** [comment]
Required for multi-container setup.

File: drupal/islandora/include/admin.form.inc [hunk 1]
```diff
@@ -59,3 +67,26 @@ function islandora_config_form_validate_stomp_url(array $element, array &$form_s
...
+ * @param array $form
+ *   Drupal renderable array.
+ */
+function islandora_config_form_validate_karaf_url(array $element, array &$form_state, array $form) {
+  // Try to connect to the server.
+  $url = $element['#value'] + '/cxf';
+  $handle = curl_init($url);
+  curl_setopt($handle,  CURLOPT_RETURNTRANSFER, TRUE);
+  $response = curl_exec($handle);
+  $httpCode = curl_getinfo($handle, CURLINFO_HTTP_CODE);
```
Comment (whikloj) [inline drupal/islandora/include/admin.form.inc]: Travis error - could you change to ```$http_code```

File: drupal/islandora/include/admin.form.inc [hunk 2]
```diff
@@ -59,3 +67,26 @@ function islandora_config_form_validate_stomp_url(array $element, array &$form_s
...
+  if($httpCode != 200) {
```
Comment (whikloj) [inline drupal/islandora/include/admin.form.inc]: Travis error - could you add a space after the ```if```, also change the ```$httpCode``` to match the change above.

**Comment by whikloj** [comment]
This is related to https://github.com/Islandora-CLAW/CLAW/issues/151 for the 7.x-2.x branch.

File: drupal/islandora/include/admin.form.inc [hunk 3]
```diff
@@ -59,3 +67,26 @@ function islandora_config_form_validate_stomp_url(array $element, array &$form_s
...
+ *   Drupal renderable array.
+ */
+function islandora_config_form_validate_karaf_url(array $element, array &$form_state, array $form) {
+  // Try to connect to the server.
+  $url = $element['#value'] + '/cxf';
+  $handle = curl_init($url);
+  curl_setopt($handle,  CURLOPT_RETURNTRANSFER, TRUE);
+  $response = curl_exec($handle);
+  $http_code = curl_getinfo($handle, CURLINFO_HTTP_CODE);
+  if($http_code != 200) {
```
Comment (DiegoPino) [inline drupal/islandora/include/admin.form.inc]: @whikloj, there are some coding standard issues with this "if"

Comment (whikloj) [inline drupal/islandora/include/admin.form.inc]: One more @nigelgbanks. Can you add in a space between the ```if``` and the opening bracket please. Thanks.

**Pull request closed without merging**